python/lsst/ts/imsim/utils/utility.py

```diff
@@ -182,3 +197,106 @@ def __exit__(self, exc_type: None, exc_val: None, exc_tb: None) -> None:
                 os.environ[key] = self._originals[key]
             else:
                 del os.environ[key]
+
+
+def get_cam_type(inst_name: str) -> CamType:
+    """Get the camera type from instrument name.
+
+    Parameters
+    ----------
+    inst_name : str
+         Instrument name.
+
+    Returns
+    -------
+    enum 'CamType'
+        Camera type.
+
+    Raises
+    ------
+    ValueError
+        Instrument name is not supported.
+    """
+    if inst_name == "lsst":
+        return CamType.LsstCam
+    elif inst_name == "lsstfam":
+        return CamType.LsstFamCam
+    elif inst_name == "comcam":
+        return CamType.ComCam
+    else:
+        raise ValueError(f"Instrument name ({inst_name}) is not supported.")
+
+
+def zernike_annular_fit(
```

Comment:
I think you can replace this with `lsst.ts.wep.utils.zernikeUtils.zernikeFit`

Reply:
I wanted to but `zernikeFit` only gives Z4 and up and the closed loop code need Z1,2,3. Maybe if we add an optional `z_start` parameter in the `zernikeFit` and `zernikeEval`?